scipy/spatial/transform/rotation.py

```diff
@@ -1463,6 +1464,73 @@ def magnitude(self):
         else:
             return angles
 
+    def reduce(self, left=None, right=None, return_best=False):
+        """Reduce this rotation with the provided rotation groups.
+
+        Parameters
+        ----------
+        left : `Rotation` instance
+            Object containing the left rotation group.
+        right : `Rotation` instance
+            Object containing the right rotation group.
+        right : bool
+            Whether to return the best indices
+
+        Returns
+        -------
+        reduced : `Rotation` instance
+            Object containing reduced rotations.
+        best_indices: integer array
+            Best indices of `left` and `right`
+        """
+        if left is None and right is None:
+            return cls(self._quat, normalized=True, copy=True)
+        elif right is None:
+            right = Rotation.from_quat([[0, 0, 0, 1.]])
```

Comment:
We definitely need to introduce `Rotation.identity`.

Reply:
I agree. I have added an `identity` method.